parsers/robot.c

```diff
@@ -42,7 +43,7 @@ static void whitespaceSwap (vString *const s)
             toReplace = '_';
         }
 
-        for(int i=0; i < vStringLength(s); i++)
+        for(i=0; i < vStringLength(s); i++)
```

Comment:
I thought so at first too, but the mingw build failed and pointed to this loop. So I greped through the code and saw everything else was c89, changed my code and then all the continuous integration checks passed.